app/src/main/res/values-fr-rFR/strings.xml

```diff
@@ -0,0 +1,242 @@
+<resources>
+    <string name="app_name" translatable="false">LibreSudoku</string>
+    <string name="nav_bar_statistics">Statistiques</string>
+    <string name="nav_bar_home">Accueil</string>
+    <string name="nav_bar_more">Plus</string>
+    <string name="new_game">Nouveau jeu</string>
+    <string name="dialog_new_game">Jeu en cours</string>
+    <string name="dialog_new_game_text">Vous avez une partie inachevé, voulez-vous vraiment en commencer une nouvelle?</string>
+    <string name="dialog_new_game_positive">Démarrer une nouvelle partie</string>
+    <string name="action_cancel">Annuler</string>
+    <string name="action_confirm">Oui</string>
+    <string name="action_play">Jouer</string>
+    <string name="action_continue">Continuer</string>
+    <string name="action_delete">Supprimer</string>
+
+    <string name="difficulty_unspecified">Non spécifié</string>
+    <string name="difficulty_simple">Simple</string>
+    <string name="difficulty_easy">facile</string>
```

Comment:
Should it start with lowercase?

Reply:
No, it's a mistake, thanks for pointing it out, it's corrected